_posts/2017-10-23-call-for-candidates-joint-board.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+subheadline: "Call for Joint Carpentries Leadership"
+title: "Call for Nominations to Joint Board"
+teaser: "Stand for election to the joint board of the merged Carpentries organization"
+header:
+ image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+   - blog
+comments: true
+show_meta: true
+authors: ["Karen Cranston"]
+
+---
+
+As most of you know, Software Carpentry and Data Carpentry and merging into a
+new organization, provisionally called "The Carpentries". This new organization
+will officially begin on January 1, 2018, with a hugely talented staff and
+dedicated community already in place.
+
+This is an exciting time, and we are looking for people who want to  help direct
+the new organization by being an elected member of the Board of Directors of The
+Carpentries. The board will include both appointed and elected members in order
+to balance community engagement with needed expertise in leading a growing
+non-profit organization. For more information about the responsibilities and
+composition of the board, see [this
+issue](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC2.md), part
+of the merger RFC.  
+
+If you are a qualified instructor who has taught at least twice in the past two
+years, or have done non-teaching work for the community, you can both stand for
+election and vote. Please visit the [list of current SCF
+members](https://software-carpentry.org/scf/members/) to see who is eligible to
+stand and vote in our election. We are updating this list to reflect Data
```

Comment:
I'm not sure the website list is current. I don't think we take people down, so it could include people who haven't taught recently. Probably the safer thing to say is:

If you're not sure if you're a member, log in to AMY (we can add a link on how to do this) and see if records show that you have taught in the last two years. If you need records updated or have any questions, please email team@carpentries.org. If you have taught workshops that aren't registered, please include a link to those workshops. We'll also be sending out an email to each instructor with their status.